Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md

```diff
@@ -293,3 +293,6 @@ spec:
        sum(rate(ceph_rgw_put[2m])) # promethues query used for autoscaling
      threshold: "90"
 ```
+
+!!! warning
+    During reconciliation of a `CephObjectStore`, the Rook Operator will reset the replica count for RGW which was set by horizontal pod scaler. The horizontal pod autoscaler will change the again once it re-evaluates the rule. This can result in a performance hiccup of several seconds after a reconciliation. This is briefly discussed (here)[https://github.com/rook/rook/issues/10001]
```

Comment:
can you point to the value that will cause the change?
also, is this going to cause scaling down of the RGWs, or only scaling up?

Reply:
@yuvalif : Sorry I don't fully understand your question about `what value`. When `cephobjectstore` reconciles rgw deployment will be configured with settings from `cephobjectstore` CR, normally the replica count for the deployment is 1. If HPAs increase the `replica count` of rgw deployment for scaling and when reconcile happens this will set back to 1 and IMO k8s reduce the pod count to make par with deployment. HPA itself has information about what is minimum/desired/maximum state. If it sees the replica value in the deployment decreased from desire then it will increase the value and again k8s will increase the pod in the deployment.

Regarding scaling down I didn't experiment with that. Here most likely we have more replicas in beginning and HPAs minimum should be less than the default value. In theory it should work similar to scale up IMO